settings/src/components/svn-password.scss

```diff
@@ -0,0 +1,5 @@
+.wporg-2fa__svn-password {
+	.wporg-2fa__svn-copy-password {
+		cursor: pointer;
+	}
+}
```

Comment:
Can we add some simple `code` styling?
```
code {
        display: inline-block;
        padding: 0 3px;
        background: var(--wp--preset--color--light-grey-2, #f6f6f6);
        border-radius: 2px;
}

Reply:
I added this to the SVN passwords styles, I guess it's probably worth adding at a higher level though..